ios/brave-ios/Tests/BraveWalletTests/AssetDetailStoreTests.swift

```diff
@@ -304,6 +540,26 @@ class AssetDetailStoreTests: XCTestCase {
     let solTxManagerProxy = BraveWallet.TestSolanaTxManagerProxy()
     let swapService = BraveWallet.TestSwapService()
 
+    let mockBtcBalance: Double = 0.001
+    let btcBalanceSatoshi =
+      formatter.weiString(
+        from: mockBtcBalance,
+        radix: .decimal,
+        decimals: Int(BraveWallet.BlockchainToken.mockBTCToken.decimals)
+      ) ?? ""
+    let bitcoinWalletService = BraveWallet.TestBitcoinWalletService()
+    bitcoinWalletService._balance = {
+      $1(
+        .init(
+          totalBalance: UInt64(btcBalanceSatoshi) ?? 0,
+          availableBalance: UInt64(btcBalanceSatoshi) ?? 0,
+          pendingBalance: 0,
+          balances: [:]
+        ),
+        nil
+      )
+    }
```

Comment:
nit: balances aren't fetched for coin market in asset detail, shouldn't need to return any BTC balance in here.